.eslintrc

```diff
@@ -18,26 +18,31 @@
     "platform_require": true
   },
   "plugins": [
+    "json",
     "mozilla",
     "react"
   ],
   "extends": [
     "eslint:recommended"
   ],
   "rules": {
-    "react/jsx-uses-react": 1,
-    "react/jsx-uses-vars": 1,
```

Comment:
Just copy-pasta'ed these `react/*` lines down a bit after the `mozilla/*` rules for maximum A-z.